hw2/03-count.js

```diff
@@ -0,0 +1,22 @@
+const input = document.getElementById("input");
+const textContainer = document.getElementById("text-container");
+
+input.addEventListener("input", () => {
+  const keyword = input.value;
```

Comment:
since we're adding this function for input, there should be an `event` passed as parameter, so I think it's better to take the value based on that event instead of taking it directly like this

this also make it more reusable in the future